tests/xtro-sharpie/watchOS-UIKit.todo

```diff
@@ -0,0 +1,2 @@
+!missing-protocol-conformance! NSAttributedString should conform to NSItemProviderReading (defined in 'NSAttributedString_ItemProvider' category)
```

Comment:
Well it's `UINSItemProvider` > `NSAttributedString_ItemProvider`, they only changed the category. Arguably `missing-protocol-conformance` shouldn't print "defined in ...".

I don't think we should move that to `.todo` we should update the `.ignore`. Or we move all the `missing-protocol-conformance` for NSItemProviderReading/Writing in the todo.

Reply:
It does not matter, everything in the `.todo` will be processed before releasing.
What matters is to have a green bump PR since it holds further beta 5 PR updating the various frameworks.